spec/spec_helper.rb

```diff
@@ -15,8 +15,7 @@
 require_relative "./helpers/user"
 
 def postgres_now
-  now = ActiveRecord::Base.connection.execute("SELECT NOW();")[0]["now"]
-  Time.parse(now)
+  ActiveRecord::Base.connection.execute("SELECT NOW();")[0]["now"]
```

Comment:
This isn't required in the later version of activerecord, as it understands these postgres native types